stardog_server.py

```diff
@@ -85,7 +85,11 @@ def default(self, *args, **kwargs):
         )
         apiuri = apiuri_parsed.geturl()
 
-        content = cherrypy.request.body.read()
+        content = None
```

Comment:
I think when the body is None and the request method is put, something was fucking up on this line - `pass`ing past any error and letting the upstream server handle it seems fine to me